color/p5js/colored-dots/index.js

```diff
@@ -0,0 +1,43 @@
+var canvas,
+	CORNER_WIDTH,
+	CORNER_HEIGHT,
+	corners,
+	c_color,
+	cc_color;
+
+function setup() {
+
+	canvas = createCanvas(800, 800);
```

Comment:
Please make the canvas 500 × 500 px. As all the other examples are and as it is requested in the guide.